database/sql/src/main/scala/cromwell/database/slick/tables/MetadataEntryComponent.scala

```diff
@@ -345,19 +349,32 @@ trait MetadataEntryComponent {
     val wmse = dbIdentifierWrapper("WORKFLOW_METADATA_SUMMARY_ENTRY", isPostgres)
     val resultSetColumnNames = s"me.${workflowUuid}, me.${callFqn}, me.${scatterIndex}, me.${retryAttempt}, me.${metadataKey}, me.${metadataValue}, me.${metadataValueType}, me.${metadataTimestamp}, me.${metadataJournalId}"
 
-    val query = sql"""
+    val query =
+      sql"""
       SELECT #${resultSetColumnNames}
       FROM #${metadataEntry} me
       INNER JOIN (
-        #${targetCallsSelectStatement(callFqn, scatterIndex, retryAttempt)}
+       #${attemptAndIndexSelectStatement(callFqn, scatterIndex, retryAttempt, "failed")}
```

Comment:
The updated query now determines the max scatter/retry across failed tasks on each CALL_FQN (`failedCalls`) and the max scatter/retry value altogether (so completion status isn't taken into account) on each CALL_FQN (`maxCalls`). 

The idea is that if the max values line up between the `METADATA_ENTRY`, `failedCalls`, and `maxCalls` you can be certain that the latest "run" on a "CALL_FQN" was a failure and can pull that in the results.